backend/coins/coin/codes.go

```diff
@@ -40,6 +40,13 @@ const (
 	// There are some more coin codes for the supported erc20 tokens in erc20.go.
 )
 
+const (
+	// UnitBtc represents Bitcoin unit.
+	UnitBtc string = "BTC"
+	// UnitSats represents Satoshi unit.
+	UnitSats string = "SATS"
```

Comment:
I don't think `SATS` is commonly used. `sat` is commonly used I think (lower case, not plural)

Reply:
Ok!